elytron/src/main/java/org/wildfly/extension/elytron/RealmParser.java

```diff
@@ -101,7 +101,7 @@
             .addAttributes(CachingRealmDefinition.ATTRIBUTES)
             .build();
     private final PersistentResourceXMLDescription distributedRealmParser = builder(PathElement.pathElement(ElytronDescriptionConstants.DISTRIBUTED_REALM), null)
-            .addAttribute(DistributedRealmDefinition.REALMS, AttributeParser.STRING_LIST, AttributeMarshaller.STRING_LIST)
+            .addAttributes(DistributedRealmDefinition.ATTRIBUTES)
```

Comment:
Because we are adding new attributes to the distributed realm, we actually need to introduce a new `distributedRealmParser_15_1` variable and a new `realmParser_15_1` variable that references it.

Reply:
@lvydra I've submitted a PR for the next Elytron subsystem bump so this will need to be adjusted again to introduce new `distributedRealmParser_16_0` and `realmParser_16_0` variables instead. See https://github.com/wildfly/wildfly-core/pull/5090.